src/main/java/uk/gov/hmcts/reform/sscs/service/JudicialRefDataService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.client.JudicialRefDataApi;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
+import uk.gov.hmcts.reform.sscs.model.client.JudicialRefDataUsersRequest;
+import uk.gov.hmcts.reform.sscs.model.client.JudicialUser;
+
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class JudicialRefDataService {
+
+    private final JudicialRefDataApi judicialRefDataApi;
+    private final IdamService idamService;
+
+    public String getJudicialUserFullName(String personalCode) {
```

Comment:
Do we have any unit tests covering this?

Reply:
We have a functional test that hits this in tribs but not unit, created a unit test for it now in common